packages/integration-sdk-cli/src/neo4j/neo4jUtilities.ts

```diff
@@ -50,3 +55,25 @@ export function buildPropertyParameters(propList: Object) {
 
   return propertyParameters;
 }
+
+// Start and end type helper functions.  Prepends a : to any nonempty results for
+// immediate use in a Neo4j command.
+export function getFromTypeLabel(relationship: Relationship): String {
+  if (relationship._fromType) {
+    return ':' + relationship._fromType.toString();
+  }
+  return '';
+}
+
+export function getToTypeLabel(relationship: Relationship): String {
+  if (relationship._toType) {
+    return ':' + relationship._toType.toString();
+  } else if (
+    (relationship._mapping as RelationshipMapping)?.targetEntity?._type
+  ) {
+    return (
+      ':' + (relationship._mapping as RelationshipMapping).targetEntity._type
```

Comment:
Do we actually support creating mapped relationships in Neo4j?

Reply:
Yep! Still a few questions around it (for example, we currently respect the `skipTargetCreation` flag, but I'm concerned it will cause odd looking graphs if we haven't already run the job that is responsible for the target's creation).